cmd/marker/main.go

```diff
@@ -37,6 +37,11 @@ func main() {
 		glog.Fatal("ovs-socket must be set")
 	}
 
+	if _, err := os.Stat(*ovsSocket); os.IsNotExist(err) {
+		glog.Info("Given ovs-socket %q was not found. Assuming the host does not have OVS installed. Entering eternal slumber. If OVS gets installed later, please restart this pod.")
+		select {} // Wait forever
```

Comment:
maybe create a loop with a sleep here?

Reply:
You mean to try looking for the file every X seconds? If the socket gets created on the host, will it get propagated to the mounted directory? (just if you know, otherwise, I'll try it out myself)

Reply:
Resolved, could you take a look again?